Resources/Locale/en-US/markings/reptilian.ftl

```diff
@@ -85,5 +85,8 @@ marking-LizardHornsMyrsore = Lizard Horns (Myrsore)
 marking-LizardHornsBighorn-horns_bighorn = Lizard Horns (Bighorn)
 marking-LizardHornsBighorn = Lizard Horns (Bighorn)
 
-marking-LizardHornsKoboldEars-horns_kobold_ears = Lizard Ears (Kobold)
+marking-LizardHornsKoboldEars-horns_floppy_kobold_ears = Lizard Ears (Kobold)
```

Comment:
did you mean to change this as well?

Reply:
no i did not mean to change this, sorry.
is there a way for me to edit and fix this mistake?

Reply:
alright i got it fixed, thank you for pointing that out to me!!